src/shared/logger/activation.ts

```diff
@@ -216,14 +216,21 @@ async function createLogWatcher(logPath: string): Promise<vscode.Disposable> {
         return { dispose: () => {} }
     }
 
-    const watcher = fs.watch(logPath, eventType => {
-        if (eventType === 'rename') {
+    let checking = false
+    const watcher = fs.watch(logPath, async eventType => {
+        if (checking || eventType !== 'rename') {
+            return
+        }
+        checking = true
+        const exists = await fs.pathExists(logPath).catch(() => true)
```

Comment:
Eek. We can try this to "stop the bleeding", but we should consider chokidar or fb-watchman later.

https://github.com/paulmillr/chokidar#why

> Node.js fs.watch ... Emits most changes as rename.

Reply:
Yeah I wanted to use VS Code's built-in watcher but it only works for files in a workspace folder.

Definitely need to avoid using `fs.watch` in general. Too many 'gotchas'.